bills-app/src/main/webapp/index.html

```diff
@@ -14,7 +14,7 @@
     <script src="api.js"></script> 
   </head>
 
-  <body onload="addCode()">
+  <body onload = addCode()>
```

Comment:
```suggestion
  <body onload="addCode()">
```
Can you keep the original here?